tf/modules/monitoring/platform-summary.tf

```diff
@@ -5,7 +5,7 @@ resource "google_logging_metric" "platform-summary-metrics" {
         resource.type="k8s_container"
         resource.labels.cluster_name="${var.cluster_name}"
         resource.labels.namespace_name="default"
-        labels.k8s-pod/app_kubernetes_io/component="queue"
+        labels.k8s-pod/app_kubernetes_io/component="queue-default"
```

Comment:
Not sure if it would make sense to start using a variable here. Maybe yes, as we'll soon move the job to a dedicated job, changing the name again.

Reply:
I *personally* still think this is fine to hardcode here; sure we'll move it soon but probably repeatedly after that